game-servers/trackmania-forever.md

```diff
@@ -0,0 +1,261 @@
+		<cup_nbwinners>3</cup_nbwinners>
+		<cup_warmupduration>2</cup_warmupduration>
+	</gameinfos>
+
+	<hotseat>
+		<game_mode>0</game_mode>
+		<time_limit>300000</time_limit>
+		<rounds_count>5</rounds_count>
+	</hotseat>
+
+	<filter>
+		<is_lan>1</is_lan>
+		<is_internet>1</is_internet>
+		<is_solo>0</is_solo>
+		<is_hotseat>0</is_hotseat>
+		<sort_index>1000</sort_index>
+		<random_map_order>0</random_map_order>
+		<force_default_gamemode>0</force_default_gamemode>
+	</filter>
+
+	<startindex>0</startindex>
+	<challenge>
+		<file>Campaigns\Nations\Blue\C01-Race.Challenge.Gbx</file>
+		<ident>eDgWjoKe2dT3GfoTCGCmI_qMvfk</ident>
+	</challenge>
+	<challenge>
+		<file>Campaigns\Nations\Blue\C02-Race.Challenge.Gbx</file>
+		<ident>hlRjJEZGm0yr1sT91CtdIwmqsti</ident>
+	</challenge>
+	<challenge>
+		<file>Campaigns\Nations\Blue\C03-Acrobatic.Challenge.Gbx</file>
+		<ident>c4oQLgleEPkNtehypwdYXTkmVvi</ident>
+	</challenge>
+	<challenge>
+		<file>Campaigns\Nations\Blue\C04-Race.Challenge.Gbx</file>
+		<ident>yWy7ROt2lgk2zL44HKdBgUjuthi</ident>
+	</challenge>
+	<challenge>
+		<file>Campaigns\Nations\Blue\C05-Endurance.Challenge.Gbx</file>
+		<ident>UR7xWwTkMeFB2kqVLVVOGDBCKFb</ident>
+	</challenge>
+	<challenge>
+		<file>Campaigns\Nations\Blue\C06-Speed.Challenge.Gbx</file>
+		<ident>fwj7Gn1nSQ_8qx6MPUtzAfHngTj</ident>
+	</challenge>
+	<challenge>
+		<file>Campaigns\Nations\Blue\C07-Race.Challenge.Gbx</file>
+		<ident>PLVn84D8NoVGjidP1pLafZP8qA8</ident>
+	</challenge>
+	<challenge>
+		<file>Campaigns\Nations\Blue\C08-Obstacle.Challenge.Gbx</file>
+		<ident>Hb_oIOr6Y4_I3aoMogsTPufz8hl</ident>
+	</challenge>
+	<challenge>
+		<file>Campaigns\Nations\Blue\C09-Race.Challenge.Gbx</file>
+		<ident>9MOwoNkpYZhw8e99cxFI3hVZrvi</ident>
+	</challenge>
+	<challenge>
+		<file>Campaigns\Nations\Blue\C10-Acrobatic.Challenge.Gbx</file>
+		<ident>XYiTfAdultrTWVJpjl_Bdnf7x4l</ident>
+	</challenge>
+	<challenge>
+		<file>Campaigns\Nations\Blue\C11-Race.Challenge.Gbx</file>
+		<ident>npRjhClGPZMs_YK_T5yUrnWY0q9</ident>
+	</challenge>
+	<challenge>
+		<file>Campaigns\Nations\Blue\C12-Obstacle.Challenge.Gbx</file>
+		<ident>znbgMZayw8uBByLWqc6kYsEfG6l</ident>
+	</challenge>
+	<challenge>
+		<file>Campaigns\Nations\Blue\C13-Race.Challenge.Gbx</file>
+		<ident>qxHe8iBNC2soNhkOoOvEKDkxZ58</ident>
+	</challenge>
+	<challenge>
+		<file>Campaigns\Nations\Blue\C14-Endurance.Challenge.Gbx</file>
+		<ident>uuGCAivChymPBU6TAHp6qIKSoR4</ident>
+	</challenge>
+	<challenge>
+		<file>Campaigns\Nations\Blue\C15-Speed.Challenge.Gbx</file>
+		<ident>n4QZfCzSzwMxsY2ILHFUEEipjtg</ident>
+	</challenge>
+</playlist>
+```
+
+## Deeplink Support
+
+Deeplinking is supported to make your server easier accessable for players. 
+
+`tmtp://#join=IPAddress:Port`
+
+## Testing Rechability of Internetserver
+
+Can be tested on http://dedimania.net/deditest
+
+## Source
+
+More Information about Serverhosting can be aquired on https://server.xaseco.org/
+
+## Game Modes
+
+TBW
```

Comment:
do you want to finish this part before merging this to the docs ?

for me in general this looks good.

Reply:
At the moment no. It was planned to explain the different game modes like I did in Xonotic. I’m not sure how long it will take until I find time.

Before the merge is getting blocked I think it should be fine to merge the configuration part. Other docs of games are WIP as well 😄